packages/block-library/src/embed/embed-loading.native.js

```diff
@@ -1,8 +1,36 @@
+/**
+ * External dependencies
+ */
+import { ActivityIndicator, View, Platform, PlatformColor } from 'react-native';
+
 /**
  * WordPress dependencies
  */
-import { Spinner } from '@wordpress/components';
+import { usePreferredColorSchemeStyle } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import styles from './styles.scss';
+
+const EmbedLoading = () => {
+	const style = usePreferredColorSchemeStyle(
+		styles[ 'embed-preview__loading' ],
+		styles[ 'embed-preview__loading--dark' ]
+	);
+
+	const activityIndicatorColor = Platform.select( {
+		ios: '#999999', // default color on iOS
+		// ActivityIndicator is blank and does not show up unless explicit color prop is provided.
+		// See https://github.com/facebook/react-native/pull/30057
+		android: PlatformColor( '?attr/colorControlActivated' ),
+	} );
```

Comment:
Not for this PR but I'm thinking that it would be nice if we could have this component generic because we most likely require it in future components.

Reply:
That's a good idea, other places where `ActivityIndicator` is used could have the same issue on Android already. I've created an issue for this: https://github.com/wordpress-mobile/gutenberg-mobile/issues/3708